### DISCUSSION TITLE
prevent duplicate entries

### DISCUSSION THREAD

**Discussion by Sagestum** [Budibase/budibase#10754] [Help] [answered]
Hi there, 

what is the best way to prevent duplicate entries in Budibase?

I have a text field in my database and I would like it not to be possible to enter this text twice.

Would this be possible as an action on a submit button? So like "check if entry already exists and if fail"?

Thanks a lot 
Best.
Chris

**Comment by melohagan** [comment, marked as answer]
Hey @Sagestum 

Related discussion: https://github.com/Budibase/budibase/discussions/6294
